site/content/entry/yuzu-progress-report-jun-2023/index.md

```diff
@@ -0,0 +1,735 @@
+
+Continuing the effort of reaching feature parity with the desktop builds, [german77](https://github.com/german77) added support for {{< gh-hovercard "10705" "installing updates and DLCs" >}} to NAND, just like on PC.
+
+{{< imgs
+	"./dlc.png| Remember to hold to select multiple files!"
+  >}}
+
+AbandonedCart once again comes to the rescue by adding support to allow this option to {{< gh-hovercard "10794" "install multiple files" >}} at once.
+It’s not fun having to constantly repeat the process to install every single thicc `Atelier Ryza: Ever Darkness & the Secret Hideout` DLC, but it’s so worth it.
+
+Big Boss [bunnei](https://github.com/bunnei) made some changes to the {{< gh-hovercard "10746" "default graphics settings" >}} to improve the out-of-the-box experience, as well as add options required to solve rendering issues and/or improve performance.
+
+Transferring from the desktop releases is `Reactive Flushing`, an option that can significantly reduce performance in the name of rendering accuracy. 
+To provide the best performance, the option is disabled by default, but if you want accurate shadows in Bayonetta games, or proper save thumbnails, remember to enable it.
+
+`Force maximum clocks` is now disabled by default, since it’s not very relevant while the main bottleneck is on the CPU side of the emulation process. 
+We’ll revisit this toggle once native code execution is implemented.
+
+{{< imgs
+	"./gpu.png| Feature parity with the PC releases, little by little"
+  >}}
+
+An often forgotten feature of the Nintendo Switch is touch support. 
+While in handheld mode, some games allow you to play without controls, just with the screen. 
+Support was present since release for the Android build, but it wasn’t accurate–the touch event would happen offset to the side of where the user would touch their finger. german77 fixed this {{< gh-hovercard "10751" "touchy behaviour," >}} making playing visual novels with no overlay buttons a joy.
+
+One of the nice additions of Android 13 is the option to set {{< gh-hovercard "10760" "per-app languages," >}} so properly declaring the available language translations allows users to take advantage of this feature. 
+Thanks [flTobi](https://github.com/FearlessTobi)!
+
+{{< imgs
+	"./lang.png| Your phone knows more languages than you"
+  >}}
+
+{{< gh-hovercard "10808" "Expanding the Debug section" >}} a bit, t895 added the option to disable `CPU Fastmem`.
+This will reduce performance, but is useful for debugging purposes.
+
+{{< imgs
+	"./debug.png| Extremely recommended to leave at default values"
+  >}}
+
+Additionally, an option to select the audio backend was added too.
+
+{{< imgs
+	"./audio.png| Volume controls"
+  >}}
+
+Because the option is only supported on Qualcomm devices, t895 {{< gh-hovercard "10864" "now hides" >}} the `Install GPU driver` option if the user is running a Mali or Intel/AMD x86 device.
```

Comment:
```suggestion
Because it is only supported on Qualcomm devices, t895 {{< gh-hovercard "10864" "now hides" >}} the `Install GPU driver` option if the user is running a non-Qualcomm device.
```